lms/djangoapps/verify_student/urls.py

```diff
@@ -139,4 +139,12 @@
         views.submit_photos_for_verification,
         name="verify_student_submit_photos"
     ),
+    # Endpoint for in-course reverification
+    # Users are sent to this end-point from within courseware
+    # to re-verify their identities by re-submitting face photos.
+    url(
+        r'^reverify/{course_id}/(?P<checkpoint_name>\w+)/$'.format(course_id=settings.COURSE_ID_PATTERN),
+        views.InCourseReverifyView.as_view(),
+        name="verify_student_incourse_reverify"
+    )
```

Comment:
Minor: Add a trailing comma here so that the next person to add a new URL will have a cleaner diff.